docker: tvm.ci_cpu docker image fails to build

The ci_cpu docker image currently fails to build, issue appears to be in:

Step 15/29 : RUN bash /install/ubuntu_install_rust.sh

Which results in 12 build errors, first of which is:

   Compiling tokio-service v0.1.0
error[E0412]: cannot find type `NonZeroI8` in module `num`
    --> /opt/rust/registry/src/github.com-1ecc6299db9ec823/serde-1.0.98/src/de/impls.rs:2416:5
     |
2416 |     NonZeroI8,
     |     ^^^^^^^^^ help: a struct with a similar name exists: `NonZeroU8`

--- a/docker/Dockerfile.ci_cpu
+++ b/docker/Dockerfile.ci_cpu
@@ -43,7 +43,6 @@ COPY install/ubuntu_install_rust.sh /install/ubuntu_install_rust.sh
 RUN bash /install/ubuntu_install_rust.sh
 ENV RUSTUP_HOME /opt/rust
 ENV CARGO_HOME /opt/rust
-ENV RUSTC_WRAPPER sccache
 
 # AutoTVM deps
 COPY install/ubuntu_install_redis.sh /install/ubuntu_install_redis.sh

--- a/docker/install/ubuntu_install_rust.sh
+++ b/docker/install/ubuntu_install_rust.sh
@@ -28,7 +28,6 @@ export CARGO_HOME=/opt/rust
 curl -s -S -L https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path --default-toolchain nightly-2019-03-24
 . $CARGO_HOME/env
 rustup component add rustfmt
-cargo install sccache --no-default-features
 
 # make rust usable by all users
 chmod -R a+w /opt/rust

